pkg/operator/ceph/controllers/nodedrain/reconcile.go

```diff
@@ -0,0 +1,119 @@
+package nodedrain
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rook/rook/pkg/operator/ceph/controllers/controllerconfig"
+
+	"github.com/coreos/pkg/capnslog"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var log = capnslog.NewPackageLogger("github.com/rook/rook", "nodedrain-controller")
+
+const (
+	nodeHostNameKey = "kubernetes.io/hostname"
+)
+
+// ReconcileNode reconciles ReplicaSets
+type ReconcileNode struct {
+	// client can be used to retrieve objects from the APIServer.
+	scheme  *runtime.Scheme
+	client  client.Client
+	log     *capnslog.PackageLogger
+	options *controllerconfig.Options
+}
+
+// Implement reconcile.Reconciler so the controller can reconcile objects
+var _ reconcile.Reconciler = &ReconcileNode{}
+
+// Reconcile reconciles a node and ensures that it has a drain-detection deployment
+// attached to it.
+// The Controller will requeue the Request to be processed again if an error is non-nil or
+// Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
+func (r *ReconcileNode) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+
+	log.Debugf("reconciling node: %v", request.Name)
+
+	node := &corev1.Node{ObjectMeta: metav1.ObjectMeta{Name: request.Name}}
+
+	err := r.client.Get(context.TODO(), request.NamespacedName, node)
+	if err != nil {
+		return reconcile.Result{}, fmt.Errorf("Could not get node %v", request.NamespacedName)
```

Comment:
`fmt.Errorf("could not get node %+v", request.NamespacedName)`

Shouldn't this be `%s`?